edweiss-app/components/lectures/remotecontrol/modal.tsx

```diff
@@ -148,3 +151,204 @@ const TwoLangsSelection: React.FC<TwoLangsSelectionProps & LightDarkProps> = ({
         </TView>
     );
 };
+
+
+// ------------------------------------------------------------
+// ----------      Modal For the timer settings      ----------
+// ------------------------------------------------------------
+
+
+export const TimerSettingModal: ReactComponent<{
+    modalRef: React.RefObject<BottomSheetModalMethods>;
+    currentTimer: number,
+    currentRecall: number
+    setTimer: React.Dispatch<React.SetStateAction<number>>;
+    setRecall: React.Dispatch<React.SetStateAction<number>>;
+    setIsCritical: React.Dispatch<React.SetStateAction<boolean>>;
+    onClose: () => void;
+}> = ({ modalRef, currentTimer, currentRecall, setTimer, setRecall, setIsCritical, onClose }) => {
+
+    // Coefficient Constants
+    const secondsCoef = 1;
+    const minutesCoef = 60 * secondsCoef;
+    const hoursCoef = 60 * minutesCoef;
+
+    // Hooks
+    const [tmpTimer, setTmpTimer] = useState(currentTimer);
+    const [tmpRecall, setTmpRecall] = useState(currentRecall);
+    const [curCoef, setCurCoef] = useState(minutesCoef);
+    const [butSelcted, setButSelcted] = useState(1);
+
+
+    // Pretty Printers
+    const ppSeconds = (seconds: number) => {
+        return `0${seconds % 60}`.slice(-2);
+    }
+
+    const ppMinute = (seconds: number) => {
+        const minutes = Math.floor(seconds / 60);
+        return `0${minutes % 60}`.slice(-2);
+    }
+
+    const ppHour = (seconds: number) => {
+        return `0${Math.floor(seconds / 3600)}`.slice(-1);
+    }
+
+    // Timer Seter interface 
+    const setTimerInterf = (seconds: number) => {
+        if (0 <= seconds && seconds <= 35999) {
```

Comment:
Here, instead of using `<= 35999` you can use your local constant `secondsInHour`like this : `< 10*secondsInHour`

Reply:
Yes introducing a constant is in order,
Just to clarify it's not  `10*secondsInHour` it's the maximum representable value in the format `X:XX:XX`.